examples/terraform/aws/main.tf

```diff
@@ -155,9 +197,11 @@ data "talos_machine_configuration" "controlplane" {
   machine_secrets  = talos_machine_secrets.this.machine_secrets
   docs             = false
   examples         = false
-  config_patches = [
-    yamlencode(local.common_machine_config_patch)
-  ]
+  config_patches = concat(
+    local.config_patches_common,
```

Comment:
Shouldn't all patches be run through the yamlencode or am I missing something?

Reply:
Oh, nvm, I see the yamlencode is specific just for registering kubelet with fqdn